README.md

```diff
@@ -129,6 +129,10 @@ Here are the different arguments:
    - **broadcastifySystemId** - *(Optional)* System ID for Broadcastify Calls (this is an integer, and different from the RadioReference system ID)
    - **audioArchive** - should the recorded audio files be kept after successfully uploading them. The options are *true* and *false* (without quotes). The default is *true*.
    - **callLog** - should a json file with the call details be kept after successful uploads. The options are *true* and *false* (without quotes). The default is *true*.
+    - **dailyLog** - should daily log files with call details be created. The options are *true* and *false* (without quotes). The default is *false*. \
+    Format is `start time,call length,recording length,talkgroup,emergency,priority,duplex,mode,` \
+    `source[|source],frequency|samples|errors|spikes[,frequency|samples|errors|spikes]`. \
```

Comment:
```
+    Format is `start time,call length,recording length,talkgroup,emergency,priority,duplex,mode,` \
+    `source[|source],frequency|samples|errors|spikes[,frequency|samples|errors|spikes]`. \
                               ^                     ^          ^
                               [?                    ]?         [?
```

I see some misbalanced bracketing and possibly a missing one too. Carets my emphasis.

Could you also document what the purpose of this daily log is, and where it's written to?

Reply:
I see having read the changes that there's no use of any `[` `]` characters, and rather that they were meant to show "optional" additions to the log line. Apologies for the confusion, I've striked out that part in my previous comment.

It is possible that these fields can be of size 0 (e.g. #471 which applies to conventional recorders), creating log lines that could look like:

```
start time,call length,recording length,talkgroup,emergency,priority,duplex,mode,,
```

I think possible users of this feature might find the documentation confusing, but I can't think of a way to describe the format. Maybe...

```
+    Format is `start time,call length,recording length,talkgroup,emergency,priority,duplex,mode,` \
+    `source|...|source,frequency|samples|errors|spikes,...,frequency|samples|errors|spikes`. \
```

But I don't like that either.

Reply:
Also moving the second bit from the original comment down here: Could you also document what the purpose of this daily log is, and where it's written to?

Reply:
I'm going to note which parts can be repeated separately. I'm guessing I can't format any of the text since I have it in a code snippet.

And good point about mentioning where it's put. As for purpose, it's an alternative to individual JSON files. Ideally, it would be a 3-way setting with the option to not write the JSON file at all (I hard coded this for the system I'm not uploading).